bin/lib/zopen-build

```diff
@@ -255,6 +259,12 @@ processOptions()
       "-g" | "--get-source")
         getSourceOnly=true
         ;;
+      "-gp" | "--generate-pax")
+        generatePax=true
+        ;;
+      "-sym" | "--symlink")
```

Comment:
should we flip the sym logic around so the default is to generate a symlink vs not to?

Reply:
This change might have an impact in the CICD logic because it also creates a symlink after the zopen build command, I'll take a closer look tomorrow

Reply:
Ok, I've made the updates, the cicd logic should be ok with this change